plugins/htpasswd/.gitignore

```diff
@@ -0,0 +1,11 @@
+npm-debug.log
+.DS_Store
+package-lock.json
+_storage/
+lib/
```

Comment:
I agree that each project has own settings, so they can have their own `.gitignore` with the necessary rules they need to add (by example, lockfiles or node_modules are not necessary because the subproject doesn't contains own lockfiles and the node_modules are defined in an upper scope

Reply:
what do you think about `dist/` folder for all the build outputs?

Reply:
`build/` and `dist/` folders shouldn't go to the repository, so add both